SnakesAndLadder/src/models/Color.java

```diff
@@ -0,0 +1,8 @@
+package models;
+
+public enum Color {
```

Comment:
What if we need more than 4 colors? This class is not scalable, can we make it better?